src/app/lightning/route-validator.ts

```diff
@@ -2,7 +2,9 @@ import { BadAmountForRouteError } from "@domain/errors"
 
 export const RouteValidator = (rawRoute: RawRoute): RouteValidator => {
   const validate = (amount): true | ApplicationError => {
-    if (amount !== rawRoute.tokens) return new BadAmountForRouteError()
+    if (amount !== rawRoute.tokens) {
+      return new BadAmountForRouteError(`${amount} !== ${rawRoute.tokens}`)
```

Comment:
Up to you, but you may want to clarify which value is which `Amount ${amount} !== Route ${rawRoute.tokens}`